src/shared/utils/challenge-listing/buckets.js

```diff
@@ -40,7 +40,7 @@ export function getBuckets(userHandle) {
       name: 'My Challenges',
       sorts: [
         SORTS.MOST_RECENT,
-        SORTS.TIME_TO_SUBMIT,
```

Comment:
Not sure, why do we need this change (see also further comments).

Reply:
@birdofpreyru The "open for registration" listing was already using SORTS.PHASE_END_TIME, which is the sort that I modified to fix the sort bug; since "my challenges" should be sorted the same way, I just updated it to use the same sort. The SORTS.TIME_TO_SUBMIT sort only sorts by final submission end time, while SORTS.PHASE_END_TIME sorts by end of next submission phase, including next checkpoints.